monitoring/opentelemetry-reactive/src/test/java/io/quarkus/ts/opentelemetry/reactive/OpenShiftGrpcIT.java

```diff
@@ -0,0 +1,42 @@
+package io.quarkus.ts.opentelemetry.reactive;
+
+import io.fabric8.kubernetes.api.model.Container;
+import io.fabric8.kubernetes.api.model.GRPCAction;
+import io.fabric8.kubernetes.api.model.Pod;
+import io.fabric8.kubernetes.api.model.Probe;
+import io.quarkus.test.bootstrap.inject.OpenShiftClient;
+import io.quarkus.test.scenarios.OpenShiftDeploymentStrategy;
+import io.quarkus.test.scenarios.OpenShiftScenario;
+import io.quarkus.test.scenarios.annotations.DisabledOnQuarkusVersion;
+import jakarta.inject.Inject;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+@OpenShiftScenario(deployment = OpenShiftDeploymentStrategy.UsingOpenShiftExtension)
+@DisabledOnQuarkusVersion(version = "2\\..*", reason = "GRPC probes are used only in Quarkus 3")
+public class OpenShiftGrpcIT extends OpenTelemetryGrpcIT {
+
+    @Inject
+    static OpenShiftClient oc;
+
+    @Test
+    void check() {
+        List<Pod> pods = oc.podsInService(app);
+        Assertions.assertEquals(1, pods.size());
+        List<Container> containers = pods.get(0).getSpec().getContainers();
+        Assertions.assertEquals(1, containers.size());
+        Container container = containers.get(0);
+        validateProbe(container.getLivenessProbe());
+        validateProbe(container.getReadinessProbe());
+    }
+
+    private static void validateProbe(Probe probe) {
+        Assertions.assertNotNull(probe);
+        GRPCAction grpcAction = probe.getGrpc();
+        Assertions.assertNotNull(grpcAction);
+        Assertions.assertEquals("io.quarkus.example.PongService", grpcAction.getService());
+        Assertions.assertEquals(9000, grpcAction.getPort());
```

Comment:
(optional) I think using `io.quarkus.test.services.quarkus.QuarkusApplicationManagedResourceBuilder#MANAGEMENT_PORT_DEFAULT` would make it clearer where 9000 comes from, or even better - am I wrong and 9000 is set somewhere else?

Reply:
This is not related to management interface (just coincidence). The port is set in the description[1] of readiness probe.

[1] https://github.com/quarkus-qe/quarkus-test-suite/pull/1212/files#diff-7407dd38483ba531986d67d36560575dd8da1588889f7c75020498b577230e25R29

Reply:
Ah, my bad. So 9000 is defult gRPC port https://quarkus.io/guides/grpc-service-implementation#quarkus-grpc-config-group-config-grpc-server-configuration_quarkus.grpc.server.port and (your) [1] tells Quarkus OpenShift that generated template should use that port. thanks @fedinskiy